core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/node-exporter.yml

```diff
@@ -106,8 +109,8 @@
       unarchive:
         remote_src: true
         src: "{{ download_directory }}/{{ exporter_file_name }}"
-        dest: "/opt/node_exporter"
-        creates: "/opt/node_exporter/node_exporter"
+        dest: /opt/node_exporter
+        creates: /opt/node_exporter/node_exporter
         extra_opts: [--strip-components=1]
         mode: '0755'
```

Comment:
What do you think about changing mode format in this PR?

Reply:
Applied